test/Jay.Tests/Tests.fs

```diff
@@ -170,4 +170,16 @@ let ``Multiline comment is skipped`` () =
             }
         */"""
     let j = Json.parse txt
-    j?test.AsBool() |> should be True
\ No newline at end of file
+    j?test.AsBool() |> should be True
+
+[<Fact(Skip = "Requires TailCAlls to be enabled")>]
+let ``Can parse deep arrays``() =
+    String.replicate 50000 "[" + String.replicate 50000 "]"
+    |> Json.parse
+    |> ignore
+
+[<Fact(Skip = "Requires TailCAlls to be enabled")>]
```

Comment:
Note:
TailCalls need to be enabled in the library build to get these tests to pass, and those are only enabled in release builds by default (FSharp.Data has them enabled in debug builds as well - see comments in https://github.com/fsprojects/FSharp.Data/pull/1411).
It also looks like you need to use XUnit v3 to be able to mark the tests as Explicit rather than just skiping them